.github/workflows/update-config.yml

```diff
@@ -0,0 +1,94 @@
+name: Pull Request Action
+permissions: write-all
+on:
+  workflow_dispatch:
+
+env:
+  GH_TOKEN: ${{ secrets.PAT_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
```

Comment:
Could be cool to authorize as the UbiquiBot instead. Here's a snippet from some old CI:

https://github.com/ubiquity/ubiquibot/blob/c8d310ec896719dc43f28cee2697bf4648ac0e5b/.github/workflows/push.yml#L29-L45

Reply:
Just checked and we still have the secret! This should work still. Would you mind adding it in here?

<img width="896" alt="image" src="https://github.com/ubiquity/ubiquibot/assets/4975670/b5bb2b01-8bca-411d-ac51-f3846406651b">